docs/adapter-sequence-diagram.md

```
@@ -221,18 +248,28 @@ CONFIG]  |   [close write channel, kill timer 3s]   |
          |   emit sessionStopped()                  |
 ```
 
+### adapter.stop timeout — fallback to process kill
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)